locales/ja/puppet.po

```diff
@@ -3,15 +3,26 @@
 # This file is distributed under the same license as the Puppet automation framework package.
 # FIRST AUTHOR <EMAIL@ADDRESS>, 2018.
 # 
+# Translators:
+# Eriko Kashiwagi <eriko.55@d5.dion.ne.jp>, 2019
+# sayaka inagaki <sayaka-inagaki@to-in.co.jp>, 2019
+# LarissaLane <larissa@puppet.com>, 2019
+# satoko inoue <satoko-inoue@to-in.co.jp>, 2019
+# Hidekatsu Tsuru <hidekatsu-tsuru@to-in.co.jp>, 2019
+# 梅田智世 <c-umeda@nifty.com>, 2019
+# Kojima Ai <ai-kojima@to-in.co.jp>, 2019
+# Erwin Hom <erwin.hom@puppet.com>, 2019
+# James Shen <James.Shen@puppet.com>, 2019
+# 
 #, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: Puppet automation framework 6.0.2-87-gd2c9723\n"
+"Project-Id-Version: Puppet automation framework 5.5.8-108-g9e04bef\n"
```

Comment:
@ehom it looks like this was based off of the 5.5.x branch instead of master (whose version would be some number of commits past the 6.1.0 tag, eg `6.1.0-39-ga81431110c`